### DISCUSSION TITLE
Guidelines on better notification content (email/sms)

### DISCUSSION THREAD

**Discussion by jakuhn** [GovAlta/ui-components#651] [Design system bugs]
Link to figma documentation:
https://www.figma.com/file/k1Z7eJlhhbAirIWEKdJxAQ/Email-Content?node-id=0%3A1

# What is being submitted?
> Proposing a collaborative effort to improve how email notification content is formatted and structured in DDI services. A place to share leanings between teams at different product maturity stages with the outcome of creating guidelines for the creation, design, and upkeep of email/sms templates.

# Why are you submitting?
> Explain why you think this should be added to or updated in the design system.

What evidence do you have that it's needed by multiple services across government?
- Most if not all of the services in the DDI or Justice Digital use some form of notification system to inform users of confirmation, updates, actions, etc. 

What evidence do you have that it meets the needs of the users of those services?
- Our team (Adjournment Digital Service) has received feedback from users with the request to improve how information is displayed in the notifications they receive, so our users can more easily access the details that are important and relevant to their needs, wants, and limitations while using ADS.

Have you checked that it doesn't already exist in the DDI Design System?
- Yes

# Have you already started or completed work on this component or pattern?
https://www.figma.com/file/k1Z7eJlhhbAirIWEKdJxAQ/Email-Content?node-id=0%3A1

# Are you currently using the proposed component within your service?
Yes, in that our service currently sends oout email and sms notification through Alberta Notify

**Comment by dlevineAB** [comment]
As part of the Notify onboarding they pointed us to these links to avoid spam filters: 
188 Spam Words to Avoid: How to Stay Out of Spam Filters [https://www.activecampaign.com/blog/spam-words](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.activecampaign.com%2Fblog%2Fspam-words&data=04%7C01%7CDavis.Levine%40gov.ab.ca%7C5d15b910c2e94a37165608d9f6161560%7C2bb51c06af9b42c58bf53c3b7b10850b%7C0%7C0%7C637811396965082293%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Ts2ufgiMQ3tdxWdNUPqHGJkXu7ex%2BpCYFjgkU%2BJvH6g%3D&reserved=0);
29 Email Deliverability Tips You Must Know in 2020 [https://blog.hubspot.com/blog/tabid/6307/bid/30594/a-marketer-s-guide-to-getting-past-email-spam-filters.aspx](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fblog.hubspot.com%2Fblog%2Ftabid%2F6307%2Fbid%2F30594%2Fa-marketer-s-guide-to-getting-past-email-spam-filters.aspx&data=04%7C01%7CDavis.Levine%40gov.ab.ca%7C5d15b910c2e94a37165608d9f6161560%7C2bb51c06af9b42c58bf53c3b7b10850b%7C0%7C0%7C637811396965082293%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Dr7%2F0QySh8CpXSQQVP2%2BsRIXwrECJe%2FvdZHNSxupq48%3D&reserved=0); and
12-Step Guide On How To Prevent Emails From Going To Spam [https://coherentpath.com/rarelogic/blog/how-to-prevent-emails-from-going-to-spam/](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcoherentpath.com%2Frarelogic%2Fblog%2Fhow-to-prevent-emails-from-going-to-spam%2F&data=04%7C01%7CDavis.Levine%40gov.ab.ca%7C5d15b910c2e94a37165608d9f6161560%7C2bb51c06af9b42c58bf53c3b7b10850b%7C0%7C0%7C637811396965082293%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=a6vee%2BrE789D8yC0MRjlI3rV8v1ZszImtFapagqZFLc%3D&reserved=0).

**Comment by dlevineAB** [comment]
We use following guidelines/patterns for our email delivery

- Provide H1 content at top of email as a simple title of what the email contains
- Only use first name in the greeting

![Screen Shot 2022-11-08 at 3 13 21 PM](https://user-images.githubusercontent.com/117781176/200687111-5970bc98-ac9a-40b6-89cf-bb8d33bfab62.png)

**Comment by ArakTaiRoth** [comment]
Closing Discussion as this is also an open Issue and I'm hoping this will close the open issue, and there doesn't appear to be anymore discussion happening here.

**Comment by ArakTaiRoth** [comment]
Discovery and design work located [here](https://goa-dio.atlassian.net/browse/DDIDS-341?atlOrigin=eyJpIjoiM2FlN2IyOWUxOTYzNDVjMGJhZjJhMDBiMTljZTU0OTgiLCJwIjoiaiJ9)

**Comment by Spark450** [comment]
This work is tracked [here](https://github.com/GovAlta/ui-components/issues/2619)
